src/api/package/packages.ts

```diff
@@ -11,6 +11,7 @@ export interface Author {
 export interface CatalogPackage {
   name: string;
   languages?: Partial<Record<Language, Record<string, unknown>>>;
+  major: number;
```

Comment:
Is this needed?

Reply:
This is actually a fix on the type - `CatalogPackage` represents data we get from the backend, so this annotation was previously missing